### ISSUE TITLE
Allow disabling the asset list report at the end of the build entirely

### ISSUE THREAD

**Issue by marcins** [parcel-bundler/parcel#6932]
# 🙋 feature request

It should be possible to entirely disable the asset output at the end of a Parcel build  without disabling other logging. I'm referring to these lines:

```
build/assets/####.js                              ⚠️  4.35 MB    1034.36s
```

## 🤔 Expected Behavior

I want to be able to disable this part of the output entirely.

## 😯 Current Behavior

Even with `--detailed-report 0` these lines still appear, and for our large build in CI it takes some 45 seconds to log all this output (based on log timestamps).

## 💁 Possible Solution

`--detailed-report 0` could mean 0 levels of output instead of no depth but still the top level assets (though this would be a breaking change to that behaviour), alternatively a separate flag for disabling this report entirely.

## 🔦 Context

As mentioned above, we have a very large build and this output adds some 45 seconds to the build time.

**Comment by mischnic** [comment]
> and for our large build in CI it takes some 45 seconds to log all this output (based on log timestamps).

That doesn't really take any time to compute (unless you have multiple hundreds of bundles?). Are you sure it's this list and not Parcel flushing the cache to disk that's taking 45 seconds?

cc @wbinnssmith

**Comment by wbinnssmith** [comment]
Yeah definitely possible @mischnic. @marcins, I think profiling is probably the best way to determine the cause.

**Comment by marcins** [comment]
I'll see what I can do in terms of profiling, as you're aware @wbinnssmith loading a profile from these builds will be challenge!

Anecdotally, while the information may be quick to compute it seems more like an output buffering issue - even when I build locally, when the build is done I can see that output streaming by for some time (whether it's the same 45s as CI, I don't know, but it's more than a second or two).

**Comment by folknor** [comment]
What I'd really like is for the code that prints the CLI bundle stats to read a config of which file types / extensions and/or subfolders to ignore, or which ones to include. I believe that is the code here: https://github.com/parcel-bundler/parcel/blob/v2/packages/reporters/cli/src/bundleReport.js#L45 ?

For example in one of my projects there's an image gallery that contains hundreds of images. I don't need to see the numbers for those images every time. They almost never change.

**Comment by folknor** [comment]
Not sure if the maintainers would want me to file a separate issue for my previous comment? Could also possibly tag this/both issue(s) with good-first-bug and other relevant tags.

**Comment by mischnic** [comment]
> I believe that is the code here: v2/packages/reporters/cli/src/bundleReport.js#L45 ?

Correct.

I think specifying patterns to filter this list is unnecessary. I think something a CLI flag like `--no-report` would solve both cases of the situations here.
